packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -443,19 +443,15 @@ export default class WalletService {
     return addr!.address
   }
 
-  public signWitness = (witness: Witness, privateKey: string, txHash: string): Witness => {
+  public signWitness = (witness: string, privateKey: string, txHash: string): string => {
```

Comment:
If it is possible to use the SDK to sign witnesses
https://github.com/nervosnetwork/ckb-sdk-js/blob/develop/packages/ckb-sdk-core/src/index.ts#L158

Reply:
updated